src/server/req_delete.c

```diff
@@ -337,6 +341,30 @@ decr_single_subjob_usage(job *parent)
 	parent->ji_qs.ji_svrflags |= JOB_SVFLG_ArrayJob; /* setting arrayjob flag back */
 }
 
+
+int update_deletejob_stat(char *jid, struct batch_request *preq, int errcode, pbs_list_head *pdelstathd)
```

Comment:
You don't need last arg as you as access to preq in 2nd arg.

Reply:
You were suggesting in an earlier comment to remove the pbs list head from batch_reply, And here you are suggesting to remove this variable also. 

then how shall we track the next pointer to add the new delete job record. ? plz explain.

Reply:
explained in https://github.com/openpbs/openpbs/pull/2063#discussion_r514255359